ast/node.go

```diff
@@ -14,17 +14,20 @@ type Node interface {
 // Statement is the interface for statement (Ex command or Comment).
 type Statement interface {
 	Node
+	stmtNode()
 }
 
 // ExCommand is the interface for Ex-command.
 type ExCommand interface {
 	Node
 	Cmd() Cmd
+	stmtNode()
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
func stmtNode is unused (U1000)